source/pele/simple_pairwise_potential.h

```diff
@@ -51,6 +81,14 @@ class SimplePairwisePotential : public BasePotential
     }
     virtual double add_energy_gradient(Array<double> x, Array<double> grad);
     virtual double add_energy_gradient_hessian(Array<double> x, Array<double> grad, Array<double> hess);
+    void get_rij(double * const r_ij, double const * const r1, double const * const r2) const
+    {
+        return _dist->get_rij(r_ij, r1, r2);
+    }
+    double get_interaction_energy_gradient(double r2, double *gij, size_t atom_i, size_t atom_j) const
```

Comment:
I agree with virtual, probably it would not work otherwise for some cases. I thought inline was handled by the compiler now, but I can add that as well.